examples/Corrupt/ormbfile.yaml

```diff
@@ -0,0 +1,2 @@
+author: "c1 <wangsiyuan.wangsy@bytedance.com>"
+format: "corruptted :)"
```

Comment:
`corruptted :)` Why we need it, is it only for UT? If so, it should not include in examples directory,  the data in the example directory are useful for user.